cmd/primaza/main.go

```diff
@@ -191,10 +235,13 @@ func getConfig() (*config, error) {
 		return nil, err
 	}
 
+	hci, _ := getRequiredEnv(EnvHealthCheckInterval)
```

Comment:
I'd propose the following change to have the value in config set to the actual value we are using for Healthcheck interval. WDYT?

```suggestion
	hci := getHealthCheckIntervalFromEnv(log)
```

and having

```
func getHealtcheckIntervalFromEnv(log logr.Logger) int {
        ehci := os.Getenv(EnvHealthCheckInterval)
	hci, err := strconv.Atoi(ehci)
	if err != nil {
		log.Info("HealthCheckInterval not set or not a integer value: using default value", "interval", cfg.HealthCheckInterval, "error", err.Error(), "default", DefaultHealthCheckInterval)
		return DefaultHealthCheckInterval
	}
	if hci <= MinimumHealtCheckInterval {
		log.Info("provided HealthCheckInterval lower than minimum: using minimum value", "interval", cfg.HealthCheckInterval, "minimum", MinimumHealtCheckInterval, "default", DefaultHealthCheckInterval)
		return MinimumHealtCheckInterval
	}

	return hci
}
```

Reply:
Done